Extensions/TextInput/JsExtension.js

```diff
@@ -200,6 +209,30 @@ module.exports = {
         .setLabel(_('Width'))
         .setGroup(_('Border appearance'));
 
+      objectProperties
+        .getOrCreate('padding')
+        .setValue((objectContent.padding || 0).toString())
+        .setType('number')
+        .setLabel(_('Padding'))
+        .setGroup(_('Border appearance'));
+
+      objectProperties
+        .getOrCreate('maxLength')
+        .setValue(objectContent.maxLength || 20)
+        .setType('number')
+        .setLabel(_('Max length'))
+        .setGroup(_('Border appearance'));
+
+      objectProperties
+        .getOrCreate('textAlign')
+        .setValue(objectContent.textAlign || 'left')
+        .setType('choice')
+        .addExtraInfo('left')
+        .addExtraInfo('right')
+        .addExtraInfo('center')
```

Comment:
I think that the order is important here. I see that the text uses the order `left, center right`, so we might as well use the same here

Reply:
This comment is still relevant